src/main/webapp/resources/js/pages/user/components/UserDetailsPage.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import {
+  Button,
+  Form,
+  Input,
+  notification,
+  Select,
+  Space,
+  Switch,
+  Typography,
+} from "antd";
+import { formatDate } from "../../../utilities/date-utilities";
+
+import {
+  useGetUserDetailsQuery,
+  useEditUserDetailsMutation,
+} from "../../../apis/users/users";
+
+/**
+ * React component to display the user details page.
+ * @returns {*}
+ * @constructor
+ */
+export default function UserDetailsPage() {
+  const { userId } = useParams();
+  const { data, isSuccess } = useGetUserDetailsQuery(userId);
```

Comment:
Rename the data to details. Here you can just do data: details. You will need to update everywhere below where you used the variable `data`

Reply:
Please see 7754873501a6615b04115435e3a3d61f11ca549a.